### PR TITLE
cnv: fixes issue #18

### PR DESCRIPTION
**Pull request by karmab**
fixes https://github.com/openshift-kni/install-scripts/issues/18
- incorporate a script for bridge handling by detecting which nic to build the bridge on top of, and making the bridge name configurable
- wait for this bridge to be properly ready on each master prior to deploying cnv
- remove kubevirt web ui check as the relevant code lives in the openshift console

this is currently done without nmstate

### CODE REVIEW

**Comment by russellb** [comment]
Is this for issue #18?  If so, can you update the PR description and commit message to reference the issue?

File: CNV/deploy-cnv.sh [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 OPERATORS_NAMESPACE="openshift-operators"
 LINUX_BRIDGE_NAMESPACE="linux-bridge"
-KWEB_UI_NAMESPACE="kubevirt-web-ui"
```
Comment (markmc) [inline CNV/deploy-cnv.sh]: These web UI changes seem unrelated - propose them as a separate PR

**Comment by e-minguez** [comment]
I think it would be better to have a single `post-install.sh` script that:

- Creates the mc
- Create any other mcs required for RHHI.Next (if any)
- Apply the mc as #28 to avoid rebooting the nodes again

**Comment by karmab** [comment]
> I think it would be better to have a single `post-install.sh` script that:
> 
> * Creates the bridge mc
> * Create any other mcs required (if any)
> * Apply the mc as #28 to avoid rebooting the nodes again

agreed, so i left the bridge script as an optional step that one might want to launch when deploying CNV rather than during post install

**Comment by e-minguez** [comment]
I believe this PR can be closed as the good one seems to be #42

**Comment by sreichar** [comment]
@karmab - any issue with closing this?

**Comment by e-minguez** [comment]
ping @karmab

**Comment by sreichar** [comment]
ping @karmab  - Should this be closed?

**Comment by karmab** [comment]
yes let s close it
